### DISCUSSION TITLE
about other function

### DISCUSSION THREAD

**Discussion by W2426566602** [xisuo67/XHS-Spider#10] [Q&A]
I think if you can get the user information through the crawler, then you can also find the corresponding XHS user information through the current application domain for quick login, as long as you log in to Xiaohongshu locally, you can directly and quickly log in to XHS-Spider; It's like logging into QQ locally and then opening wegame

**Comment by xisuo67** [comment]
手动设置cookie的目的是为了防止多次调用请求后，被封的风险，程序目前可以设置多个cookie，每次调用请求的时候会随机拿一个cookie加入请求头去调用接口。
当然有一部分原因还是因为自己太懒了，没有加入自动设置cookie的功能。后面有时间会考虑加上去。不过不在近期计划内
